langchain4j-azure-ai-search/src/main/java/dev/langchain4j/store/embedding/azure/search/AzureAiSearchEmbeddingStore.java

```diff
@@ -53,23 +53,23 @@ public class AzureAiSearchEmbeddingStore implements EmbeddingStore<TextSegment>
 
     private SearchClient searchClient;
 
-    public AzureAiSearchEmbeddingStore(String endpoint, AzureKeyCredential keyCredential, int dimensions) {
-        this.initialize(endpoint, keyCredential, null, dimensions, null);
+    public AzureAiSearchEmbeddingStore(String endpoint, AzureKeyCredential keyCredential, int dimensions, boolean setupIndex) {
```

Comment:
Instead of adding a new parameter to ignore another parameter, should we create a new ctor with only `endpoint` and `keyCredential`?
I guess this will be more useful for search/retrieval phase (instead of `new AzureAiSearchEmbeddingStore(endpoint, keyCredential, -1, false);` ?

Also, please avoid breaking existing public API.